src-built-in/components/toolbarRefactored/favorites.tsx

```diff
@@ -0,0 +1,14 @@
+// import * as React from "react";
+
+// import usePins from "../UIAPI/hooks/pins";
+// import { FinsembleToolbarSection } from "@chartiq/finsemble-react-controls";
+
+// export const Favorites: React.FunctionComponent = () => {
+//     const pinnedItems = usePins("favorites");
+//     return (
+//         // FinsembleToolbarSection needs to be refactored
+//         <FinsembleToolbarSection>
```

Comment:
There shouldn't be a FinsembleToolbarSection. This should just be a div.

Styling will need to be done at the template level and we will no longer be using anything from finsemble-react-controls.

Reply:
Wow this is actually a critical piece of information that I missed. I thought we were going to keep the elements in react control. Thanks for pointing this out. This would help with the entire Favorite design